src/classes/devices/diFluid/protocol.ts

```diff
@@ -0,0 +1,384 @@
+export namespace diFluid {
+  export enum func {
+    'Device Info',
```

Comment:
It would be good if we set the enums directly:
device info =1
device settings = 2
device action = 3 

to make sure that no issues with compiling could take place